demisto_sdk/commands/content_graph/content_graph_commands.py

```diff
@@ -65,9 +66,50 @@ def update_content_graph(
         dependencies (bool): Whether to create the dependencies.
         output_path (Path): The path to export the graph zip to.
     """
+    builder = ContentGraphBuilder(content_graph_interface)
+    if os.getenv("DEMISTO_SDK_CREATE_GRAPH"):
+        logger.info("DEMISTO_SDK_CREATE_GRAPH is set. Will create a new graph")
+        builder.create_graph()
```

Comment:
why not: if force_create: create_content_graph(args); else: [keep same logic] (instead of separating to the new _download_and_update_graph method)?

Reply:
You right, changed to this